src/authorization-control.ts

```diff
@@ -65,4 +65,39 @@ export class AuthorizationControlApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take an Authorization Control token with an optional fields
+   * argument, send those to Marqeta, and have an Authorization Control
+   * updated and returned to the caller.
+   */
+  async byTokenId(search: {
+    token?: string,
+    fields?: string[],
+  } = {}): Promise<{
+    success: boolean,
+    body?: AuthorizationControl,
+    error?: MarqetaError,
+  }> {
+    const {
+      token,
+      fields = ''
```

Comment:
If `fields` is a `string[]`... then how is it defaulted to the empty `string`? And given that `fire()` will remove empty strings, why not just say:

```typescript
const {
  token,
  fields,
} = search
```

??

Reply:
` Fire()` does not allow for `'string[] | undefined'`, therefore, compiler complains:
 
` <html>TS2322: Type 'string[] | undefined' is not assignable to type 'string | number | boolean | string[]'.<br/>Type 'undefined' is not assignable to type 'string | number | boolean | string[]'.`

Reply:
nvm, got it...